map/src/i18n/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Strings } from './iface';
+import en from './langs/en';
+import ru from './langs/ru';
+
+const STRINGS = {
+  en,
+  ru,
+};
+
+type Language = keyof typeof STRINGS;
+
+const selectedLanguage = localStorage.getItem('selectedLanguage');
+
+const queryLanguage = window.location.href.split('/')[3];
```

Comment:
This is not a good / reliable way to get the first path component of the URL. In addition, it's a bit less flexible than using the query string, as the variable indicates it is doing.